### PR TITLE
[docstring] Fix docstring for `LlamaTokenizer` and `LlamaTokenizerFast`

### PR DESCRIPTION
**Pull request by minhoryang**
# What does this PR do?

Fixes #26638 only for `LlamaTokenizer` and `LlamaTokenizerFast`.

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [x] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@ArthurZucker and @younesbelkada

### CODE REVIEW

**Comment by minhoryang** [comment]
@ydshieh ready for ✅ thanks!

**Comment by minhoryang** [comment]
@LysandreJik could you assign this to @ydshieh? Thank you.

**Comment by minhoryang** [comment]
> **[PASSED src/transformers/models/llama/tokenization_llama_fast.py::transformers.models.llama.tokenization_llama_fast.LlamaTokenizerFast](https://app.circleci.com/pipelines/github/huggingface/transformers/75029/workflows/2bc31261-2db7-4bb0-a657-fe0e69138af8/jobs/951332/artifacts)**

File: src/transformers/models/llama/tokenization_llama.py [hunk 1]
```diff
@@ -71,6 +71,42 @@ class LlamaTokenizer(PreTrainedTokenizer):
...
             Path to the vocabulary file.
+        unk_token (`str` or `tokenizers.AddedToken`, *optional*, defaults to `"<unk>"`):
+            A special token representing an out-of-vocabulary token.
+        bos_token (`str` or `tokenizers.AddedToken`, *optional*, defaults to `"<s>"`):
+            A special token representing the beginning of a sentence.
+        eos_token (`str` or `tokenizers.AddedToken`, *optional*, defaults to `"</s>"`):
+            A special token representing the end of a sentence.
+        pad_token (`str` or `tokenizers.AddedToken`, *optional*):
+            A special token used to make arrays of tokens the same size for batching purpose. Will then be ignored by
+            attention mechanisms or loss computation.
```
Comment (ydshieh) [inline src/transformers/models/llama/tokenization_llama.py:82]: Could we use the same descriptions as in `src/transformers/models/llama/tokenization_llama_fast.py`?

File: utils/not_doctested.txt [hunk 2]
```diff
@@ -626,7 +626,6 @@ src/transformers/models/lilt/configuration_lilt.py
 src/transformers/models/llama/configuration_llama.py
 src/transformers/models/llama/convert_llama_weights_to_hf.py
 src/transformers/models/llama/modeling_llama.py
-src/transformers/models/llama/tokenization_llama_fast.py
```
Comment (ydshieh) [inline utils/not_doctested.txt:629]: why we need this change ?

**Review by ydshieh — COMMENTED** [review]
Thank you for the contribution. I left a few comments to address, thank you!

File: src/transformers/models/llama/tokenization_llama_fast.py [hunk 3]
```diff
@@ -78,26 +78,28 @@ class LlamaTokenizerFast(PreTrainedTokenizerFast):
...
-            spaces.
-
-        bos_token (`str`, *optional*, defaults to `"<s>"`):
-            The beginning of sequence token that was used during pretraining. Can be used a sequence classifier token.
-
-        eos_token (`str`, *optional*, defaults to `"</s>"`):
-            The end of sequence token.
-
-        unk_token (`str`, *optional*, defaults to `"<unk>"`):
+            Whether or not to cleanup spaces after decoding, cleanup consists in removing potential artifacts like
```
[the remark this answers is not in the document]
Comment (minhoryang) [reply src/transformers/models/llama/tokenization_llama_fast.py:88]: Oh! I changed.

File: utils/not_doctested.txt (hunk 2 above)
Comment (minhoryang) [reply utils/not_doctested.txt:629]: [These lines in `src/transformers/models/llama/tokenization_llama_fast.py`](https://github.com/huggingface/transformers/pull/26669/files/1231d4188fa5ae8fe0572999fb3903cd5185c347#diff-a7c16a11921618850a92fb91c4c7d4764d1222a1c5c4553b53d1c37a6cb3780aR63-R68) can be tested after this PR merged.

Comment (ydshieh) [reply utils/not_doctested.txt:629]: oh, you are right. My head is flooded by the many `docstring` PRs ...sorry 😅

File: src/transformers/models/llama/tokenization_llama.py (hunk 1 above)
Comment (minhoryang) [reply src/transformers/models/llama/tokenization_llama.py:82]: Yes, I matched the description from `src/transformers/models/llama/tokenization_llama_fast.py`.

**Review by ydshieh — APPROVED** [review]
Super nice 🚀 💯 ! Thank you @minhoryang

**Pull request merged**